requirements-dev.txt

```diff
@@ -20,6 +20,7 @@ py==1.5.3
 pygments==2.1.3
 mock==2.0.0
 requests==2.20.0
+websocket-client==0.54.0
```

Comment:
We need to add  boto3 as well if we want to be able to run the tests for the websockets app from scratch,